ocs_ci/ocs/node.py

```diff
@@ -321,3 +322,36 @@ def get_node_resource_utilization(nodename=None, node_type='worker'):
                     'memory': int(memory_utilization)
                 }
     return utilization_dict
+
+
+def node_network_failure(node_names, wait=True):
+    """
+    Induce node network failure
+    Bring node network interface down, making the node unresponsive
+
+    Args:
+        node_names (list): The names of the nodes
+        wait (bool): True in case wait for status is needed, False otherwise
+
+    Returns:
+        bool: True if node network fail is successful
+    """
+    if not isinstance(node_names, list):
+        node_names = [node_names]
+
+    ocp = OCP(kind='node')
+    fail_nw_cmd = "ifconfig $(route | grep default | awk '{print $(NF)}') down"
+
+    for node_name in node_names:
+        try:
+            ocp.exec_oc_debug_cmd(
+                node=node_name, cmd_list=[fail_nw_cmd], timeout=15
+            )
+        except TimeoutExpired:
+            pass
+
+    if wait:
+        wait_for_nodes_status(
+            node_names=node_names, status=constants.NODE_NOT_READY
+        )
+    return True
```

Comment:
In case you will not return True, it will return None by default which is ok for calling this with assert or in 'if' statement but I suggest to add the 'return False' just to make it more readable

Reply:
@RazTamir  I'm expecting a  'CommandFailed' exception from ocp.exec_oc_debug_cmd() or 'ResourceWrongStatusException' from wait_for_nodes_status() will be raised in case something goes wrong, Otherwise the function will return True
Should we handle all exceptions and just return False

Reply:
ok. thanks for clarifying